frontend/package.json

```diff
@@ -7,33 +7,27 @@
     "test": "npm run typecheck && jest",
     "dev": "next",
     "dev:local-wamp": "env WAMP_NEAR_EXPLORER_URL=ws://localhost:8080/ws npm run dev",
-    "dev:local-wamp-with-legacy-sync": "env NEAR_EXPLORER_DATA_SOURCE=LEGACY_SYNC_BACKEND npm run dev:local-wamp",
     "dev:local-wamp-with-indexer": "env NEAR_EXPLORER_DATA_SOURCE=INDEXER_BACKEND npm run dev:local-wamp",
     "dev:local-wamp-betanet": "env 'NEAR_NETWORKS=[{\"name\": \"betanet\", \"explorerLink\": \"http://localhost:3000/\", \"aliases\": [\"localhost:3000\", \"127.0.0.1:3000\"], \"nearWalletProfilePrefix\": \"https://wallet.betanet.near.org/profile\"}]' npm run dev:local-wamp",
```

Comment:
Let's drop betanet for now to avoid confusion. There is no indexer for explorer for betanet yet.

Reply:
Do we have to do like so in `./backend/package.json` file?